tests/resources/test-types.json

```diff
@@ -13202,6 +13202,38 @@
               }
             ]
           },
+          {
+            "id": "450",
+            "sortId": "450",
+            "typeOfTest": "desk-based",
+            "linkedIds": null,
+            "name": "Mutual Recognition Cat E Statutory",
+            "testTypeName": "Mutual Recognition Cat E Statutory",
+            "forVehicleType": ["trl"],
+            "forVehicleSize": null,
+            "forVehicleConfiguration": null,
+            "forVehicleAxles": null,
+            "forEuVehicleCategory": null,
+            "forVehicleClass": null,
+            "forVehicleSubclass": null,
+            "forVehicleWheels": null,
+            "forProvisionalStatus": true,
+            "testCodes": [
+              {
+                "forVehicleType": ["trl"],
+                "forVehicleSize": ["small", "large"],
```

Comment:
Can I ask where you got this from please?

`"forVehicleSize": ["small", "large"]` 

Checking the taxonomy all the others appear to be associated with `psv` rather than trailers when it is used.

I'm curious as that is the only thing differentiating it from 420 and 438, so I wonder whether it is just as easy to add it to one of those `forVehicleType` arrays?

Reply:
My mistake, i'll remove it now

Reply:
Okay thanks, what about the question in relation to the objects. There may be a valid reason, such as reporting different vehicle types for that test type maybe, but do you know if there is a need for a new separate object, given the defaultTestCode is identical?

Reply:
I don't think that there's need for a separate object, good shout. I will add it into already existing, 420 section.